tests/MCPClient/test_transcribe_file.py

```diff
@@ -221,3 +221,41 @@ def test_main_if_fprule_is_disabled(
         ).count()
         == 0
     )
+
+
+@pytest.mark.django_db
+@mock.patch("transcribe_file.executeOrRun")
+def test_main_if_command_is_python_script(
+    _execute_or_run,
+    fpcommand,
+    sip_file,
+    task,
+    fprule_transcription,
+    file_format_version,
+):
+    _execute_or_run.return_value = (
+        0,
+        fpcommand.command,
+        "",
+    )
+    job = mock.Mock(spec=Job)
+
+    result = transcribe_file.main(job, task_uuid=task.taskuuid, file_uuid=sip_file.uuid)
+
+    assert result == 0
+
+    assert job.print_error.mock_calls == [
+        mock.call(f"Transcribing original {sip_file.uuid}")
+    ]
+    assert job.write_output.mock_calls == [mock.call(fpcommand.command)]
+
+    # executeOrRun is called once
```

Comment:
Let's use full sentences in comments.

```suggestion
    # executeOrRun is called once.
```